engineering-feedback/feedback-faq.md

```diff
@@ -0,0 +1,39 @@
+# Engineering Feedback Frequently Asked Questions (F.A.Q.)
+
+The questions below are common questions related to the feedback process. The answers are intended to help both Microsoft employees and customers.
```

Comment:
Should we say Microsoft engineers instead of Microsoft employees?

Reply:
@mcollier - what about this one?

Reply:
I like "employees" because it isn't specific to people in CSE.  The FAQ may also help members of the Microsoft account team (which encompasses multiple roles).  I feel "employees" is all encompassing.